build.gradle

```diff
@@ -37,6 +37,7 @@ buildscript {
         // Taken from https://github.com/opensearch-project/alerting/blob/main/build.gradle#L33
         // 1.0.0 -> 1.0.0.0, and 1.0.0-SNAPSHOT -> 1.0.0.0-SNAPSHOT
         opensearch_build = opensearch_version.replaceAll(/(\.\d)([^\d]*)$/, '$1.0$2')
+        buildVersionQualifier = System.getProperty("build.version_qualifier")
```

Comment:
Please refer to these changes for qualifier.
https://github.com/opensearch-project/security/pull/1738/files

Thanks.